src/main/resources/assets/viafabricplus/lang/en_us.json

```diff
@@ -47,13 +47,13 @@
   "general_settings.viafabricplus.emulate_inventory_actions_in_alpha_versions": "Emulate inventory actions in alpha versions",
 
   "bedrock_settings.viafabricplus.confirm_transfer_server_prompt": "Open prompt GUI to confirm transferring to other servers",
-  "bedrock_settings.viafabricplus.click_to_set_bedrock_account": "Click to set account for Bedrock edition",
+  "bedrock_settings.viafabricplus.click_to_set_bedrock_account": "Click to set account for Bedrock Edition",
   "bedrock_settings.viafabricplus.replace_default_port": "Replace default port in server list",
 
   "debug_settings.viafabricplus.queue_config_packets": "Queue config packets",
   "debug_settings.viafabricplus.disable_sequencing": "Disable sequencing",
   "debug_settings.viafabricplus.smooth_out_merchant_screens": "Smooth out merchant screens",
-  "debug_settings.viafabricplus.execute_inputs_in_sync": "Execute inputs in sync",
+  "debug_settings.viafabricplus.execute_inputs_in_sync": "Execute inputs in synchronization",
```

Comment:
Nope, "in sync" is an English phrase, "in synchronization" sounds weird and idk it might even be wrong.

Reply:
“synchronously” could work here, but i prefer the original

Reply:
synchronously could work without the "in", will change that myself.